### ISSUE TITLE
Initial run of generator fails

### ISSUE THREAD

**Issue by larskluge** [javereec/spree_elasticsearch#23]
`bundle exec rails g spree_elasticsearch:install` fails to run without having a valid `config/elasticsearch.yml` file in your project already (which the generator is supposed to create). Reason: the generator loads the project dependencies which of one is the settings logic piece that requires the config file to exist.

**Comment by javereec** [comment]
That is correct. It used to work, but somewhere the behaviour was changed in rails and I haven't found a way to circumvent this.

**Comment by jspizziri** [comment]
can the order of install steps be updated so that the elasticsearch.yml is added before the generator is run?

**Comment by javereec** [comment]
That is exactly what I can't find. I've really looked into other gems, but haven't found anything that does something similar. If you can find something and submit a pull request, I'm more than happy to merge it.

**Comment by jkelleyj** [comment]
FYI - this issue was still there for me if I ran the touch command to create the elasticsearch.yml file.  In rails 4.2.5, if I run the install command without that file present, it works and successfully generates the config file.
